test/integration/user_logs_in_with_facebook_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+class UserLogsInWithTwitterTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  def setup
+    Capybara.app = FeedBee::Application
+    stub_omniauth
+  end
+
+  def stub_omniauth
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+      provider: 'facebook',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.